### ISSUE TITLE
Bad font resolution in Windows

### ISSUE THREAD

**Issue by mateusk** [hironishihara/ofxTrueTypeFontUC#23]
Hi, thanks for this addon, I've been using it for some time. But recently I´m getting a very bad font rendering, text looks like in a very low resolution image. I've tried to change font DPI, change font files, no difference, can´t get it to render properly. I'm using Windows 10, OF 0.9.8. Any hints on what could be the problem?

**Comment by mateusk** [comment]
I figured out that for some weird reason the bad resolution only happens if I use an instance of ofxTrueTypeFontUC inside a class and create a pointer to that class. For example, I created a class for a button, and used ofxTrueTypeFontUC for the labels. If I create an instance of ofxTrueTypeFontUC in ofApp.cpp, the font rendering is perfect. But when I create a pointer to the class that creates a ofxTrueTypeFontUC instance, then font rendering is terrible. I guess that has something to do with how ofxTrueTypeFontUC  uses textures to render the fonts, but it would be nice to better understand the problem.
